orc8r/gateway/python/magma/directoryd/rpc_servicer.py

```diff
@@ -63,6 +71,8 @@ def UpdateRecord(self, request, context):
         Args:
             request (UpdateRecordRequest): update record request
         """
+        logging.debug("UpdateRecord request received")
```

Comment:
would it be enough to just log: logging.debug("UpdateRecord") ?

Reply:
This message is verbose in purpose: if you have grpc logging disabled you won't know if this is an outgoing or incoming message.

Also sometimes it is useful to have those references for grepping logs (differentiate between messages received and message sent)